### PR TITLE
fix #73128 sonikelf.ru

### LINKED ISSUE #73128: sonikelf.ru
### Issue URL (Ads)
 [https://sonikelf.ru/kak-nastroit-brandmauer-windows/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fsonikelf.ru%2Fkak-nastroit-brandmauer-windows%2F)
### Comment
Username: @spok1
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> AdGuard Russian

### PR DESCRIPTION
**Pull request by wout-codes**
#73128

### CODE REVIEW

File: RussianFilter/sections/specific.txt [hunk 1]
```diff
@@ -49,6 +49,7 @@ crazymailing.com,lotzon.pro,novosti-n.org,prokazan.ru,20minut.ua,alnews.com.ua,g
 ! vast ads on anime sites - for example: animego.org, yummyanime.club, etc
 /:\/\/[0-9]?[a-z0-9]{11,}(\.com|\.in\.net)/$xmlhttprequest,domain=aniqit.com|kodik.info|start.u-cdn.top|ustore.bz|u-stream.in
 !
+sonikelf.ru##nav[class^="widget_text section"]
```
Comment (zloyden) [inline RussianFilter/sections/specific.txt:52]: Too wide rule. Please exchange it to the `sonikelf.ru#?#.b-sidebar > nav.widget_text div[id^="yandex_rtb"]:upward(nav.widget_text)` -> `css_extended.txt`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply RussianFilter/sections/specific.txt:52]: What's the problem with the rule I created? May I know?

Comment (zloyden) [reply RussianFilter/sections/specific.txt:52]: As I said above, it's too wide. Can cause false-positives and we avoid such rules.

Comment (wout-codes) [reply RussianFilter/sections/specific.txt]: I see.

**Comment by zloyden** [comment]
You deleted the entire section of Russian filter. I will make necessary changes directly.

**Pull request closed without merging**